androidApp/onbording/src/main/java/tech/takahana/iconwallpaper/android/onbording/ui/screen/WelcomeScreen.kt

```diff
@@ -0,0 +1,60 @@
+package tech.takahana.iconwallpaper.android.onbording.ui.screen
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import tech.takahana.iconwallpaper.android.onbording.R
+import tech.takahana.iconwallpaper.android.onbording.ui.components.BottomButton
+import tech.takahana.iconwallpaper.android.onbording.ui.theme.IconWallPaperTheme
+
+@Composable
+fun WelcomeScreen() {
+    IconWallPaperTheme {
+        Column(
+            modifier = Modifier
+                .padding(20.dp)
+                .fillMaxSize(),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Bottom,
+        ) {
+            Image(
+                painter = painterResource(id = R.drawable.welcome_dog),
+                contentDescription = stringResource(R.string.welcome_dog),
+            )
+            Spacer(modifier = Modifier.padding(vertical = 24.dp))
```

Comment:
`Modifier.height` の方が直感的でわかりやすそうです👀

```suggestion
            Spacer(modifier = Modifier.height(height = 24.dp))
```